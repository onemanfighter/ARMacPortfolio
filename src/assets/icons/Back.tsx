import * as React from 'react';

function BackIcon(props: React.SVGProps<SVGSVGElement>) {
  return (
    <svg
      viewBox="0 0 512 512"
      fill="currentColor"
      height="1em"
      width="1em"
      {...props}
    >
      <path
        fill="none"
        stroke="currentColor"
        strokeLinecap="round"
        strokeLinejoin="round"
        strokeWidth={32}
        d="M249.38 336L170 256l79.38-80M181.03 256H342"
      />
      <path
        fill="none"
        stroke="currentColor"
        strokeMiterlimit={10}
        strokeWidth={32}
        d="M448 256c0-106-86-192-192-192S64 150 64 256s86 192 192 192 192-86 192-192z"
      />
    </svg>
  );
}

export { BackIcon };
