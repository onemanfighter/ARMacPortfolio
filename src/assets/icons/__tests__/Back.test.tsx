import { render } from '@testing-library/react';
import { BackIcon } from '../Back';

describe('Back', () => {
  it('should render correctly', () => {
    const { container } = render(<BackIcon />);

    expect(container).toMatchSnapshot();
  });
});
