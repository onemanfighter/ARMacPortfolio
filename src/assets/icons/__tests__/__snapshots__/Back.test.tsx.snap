// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Back should render correctly 1`] = `
<div>
  <svg
    fill="currentColor"
    height="1em"
    viewBox="0 0 512 512"
    width="1em"
  >
    <path
      d="M249.38 336L170 256l79.38-80M181.03 256H342"
      fill="none"
      stroke="currentColor"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="32"
    />
    <path
      d="M448 256c0-106-86-192-192-192S64 150 64 256s86 192 192 192 192-86 192-192z"
      fill="none"
      stroke="currentColor"
      stroke-miterlimit="10"
      stroke-width="32"
    />
  </svg>
</div>
`;
